src/test/scala/no/ndla/articleapi/service/converters/MathMLConverterTest.scala

```diff
@@ -40,4 +40,13 @@ class MathMLConverterTest extends UnitSuite with TestEnvironment {
     result.content should equal (expectedContent)
   }
 
+  test("nbsp in MathML <mo> tags should not be converted to space if only nbsp") {
```

Comment:
Legg gjerne inn en test i `ConverterServiceTest.scala` også. Den tester hele converter pipelinen slik at vi har mulighet for å spotte tilfeller hvor forskjellige moduler potensielt ødelegger for hverandre.